netlink-packet-route/src/rtnl/link/nlas/link_infos.rs

```diff
@@ -478,6 +488,263 @@ impl<'a, T: AsRef<[u8]> + ?Sized> Parseable<NlaBuffer<&'a T>> for InfoKind {
     }
 }
 
+// https://elixir.bootlin.com/linux/v5.9.8/source/drivers/net/bonding/bond_netlink.c#L82
+#[derive(Debug, PartialEq, Eq, Clone)]
+pub enum InfoBond {
+    Mode(u8),
+    ActiveSlave(u32),
+    MiiMon(u32),
+    UpDelay(u32),
+    DownDelay(u32),
+    UseCarrier(u8),
+    ArpInterval(u32),
+    ArpIpTarget(Vec<u8>),
+    ArpValidate(u32),
+    ArpAllTargets(u32),
+    Primary(u32),
+    PrimaryReselect(u8),
+    FailOverMac(u8),
+    XmitHashPolicy(u8),
+    ResendIgmp(u32),
+    NumPeerNotif(u8),
+    AllSlavesActive(u8),
+    MinLinks(u32),
+    LpInterval(u32),
+    PacketsPerSlave(u32),
+    AdLacpRate(u8),
+    AdSelect(u8),
+    AdInfo(Vec<u8>),
```

Comment:
This `AdInfo` has nested NLAs. `IFLA_BOND_AD_INFO_AGGREGATOR` and etc.

Reply:
Done.